src/modules/auth/webauthn/WebAuthnScreen.js

```diff
@@ -41,12 +40,26 @@ function WebAuthnCard(props) {
 	const [ isLoading, setIsLoading ] = useState(true);
 	const [ isSubmitting, setIsSubmitting ] = useState(false);
 	const [ authenticators, setAuthenticators ] = useState([]);
-	const [ editing, setEditing ] = useState(false);
+	const [ globalEditMode, setGlobalEditMode ] = useState(false);
+
+	const { handleSubmit, register, formState, setValue, resetField, getValues } = useForm();
+
+	const regName = register(
+		"name",
+		{
+			validate: {
+				emptyInput: value => (value && value.toString().length !== 0) || t("WebAuthnScreen|Key name cannot be empty!"),
+				startWithNumber: value => !(/^\d/).test(value) || t("WebAuthnScreen|Invalid format, resource cannot start with a number"),
+				startWithDash: value => !(/^[-]$/).test(value) || t("WebAuthnScreen|Invalid format, resource cannot start with a dash"),
+				startWithUnderscore: value => !(/^[_]$/).test(value) || t("WebAuthnScreen|Invalid format, resource cannot start with a underscore"),
+				validation: value => (/^[a-z][a-z0-9._-]{0,128}[a-z0-9]/).test(value) || t("WebAuthnScreen|Invalid format, only lower-case letters, numbers, dash and underscore are allowed"),
```

Comment:
@Aringoaway Rename `validation` to something more sensible that it is understood what validation is used here